gsoc/2019/mentors.md

```diff
@@ -17,13 +18,16 @@ layout: plain
 * Mihaela Gheata [mihaela.gheata@cern.ch](mailto:mihaela.gheata@cern.ch) CERN/ISS
 * Gilles Grasseau [gilles.grasseau@llr.in2p3.fr](mailto:gilles.grasseau@llr.in2p3.fr) LLR
 * Lukas Heinrich [lukas.heinrich@cern.ch](mailto:lukas.heinrich@cern.ch) CERN
+* Pierre Kesterner [ierre.kestener@cea.fr](mailto:pierre.kestener@cea.fr) CEA
```

Comment:
Since Pierre's affiliation is CEA, could you please add it as a new organization to `_gsocorgs/2019/`?

Reply:
He is now affiliated tp MdS, and I added a page in _gsocorgrs. As far as I can say, all the issues are solved.